airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java

```diff
@@ -221,6 +221,15 @@ public static Map<String, Object> flatten(final JsonNode node) {
         mergeMaps(output, field, flatten(value));
       }
       return output;
+    } else if (node.isArray()) {
+      final Map<String, Object> output = new HashMap<>();
+      final int arrayLen = node.size();
+      for (int i = 0; i < arrayLen; i++) {
+        final String field = String.format("[%d]", i);
+        final JsonNode value = node.get(i);
+        mergeMaps(output, field, flatten(value));
+      }
+      return output;
```

Comment:
thanks @adam-bloom but can you add a unit test for this? I'll request the team to review it becsause the change can impact more than just Redshift.